xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-plugins/src/main/webjar/xwiki-slash/plugin.js

```diff
@@ -344,8 +344,9 @@
           // Get the text before the caret.
           var left = text.slice(0, offset),
               // Will look for the marker followed by text.
-              match = left.match(new RegExp(escapeRegExp(config.marker) + '.{0,30}$'));
-          if (match) {
+              match = left.match(new RegExp(escapeRegExp(config.marker) + '.{0,30}$')),
+              subMatch = left.match(new RegExp(escapeRegExp(config.marker) + '.{0,30}::\\S{0,30}$'));
```

Comment:
Might be interesting to develop a bit what's going on in the comments. Looks magic.

Reply:
I'm making sure the dropdown doesn't show when we add `::` in a slash query. I will add comments, thanks.